main.js

```diff
@@ -39,6 +39,7 @@ function getMainWindow() {
 let tray = null;
 const startInTray = process.argv.find(arg => arg === '--start-in-tray');
 const usingTrayIcon = startInTray || process.argv.find(arg => arg === '--use-tray-icon');
+const importMode = process.argv.find(arg => arg === '--import');
```

Comment:
Assuming we are using these as booleans, could we use `Array::some` instead of `Array::find`?